src/utils/textTransform.ts

```diff
@@ -0,0 +1,14 @@
+import * as CSS from 'csstype';
+import { RequiredTheme, ResponsiveValue, style, Theme } from 'styled-system';
+
+export type TextTransformProps<ThemeType extends Theme = RequiredTheme> = {
+  textTransform?:
+    | ResponsiveValue<CSS.Property.TextTransform, ThemeType>
```

Comment:
![image](https://user-images.githubusercontent.com/590685/125150744-16438600-e0f7-11eb-8e83-5a3cec9a8254.png)

`CSS.Property.<Whatever>` is a type for the valid values of whatever CSS property, so that should help avoid some hardcoding on our part.

Reply:
the syntax of this export is completely opaque to me :?